pkg/virtctl/expose/expose.go

```diff
@@ -125,6 +128,17 @@ func (o *Command) RunE(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("unknown service type: %s", strServiceType)
 	}
 
+	// convert from string to IPFamily type enum
+	strIPFamily = strings.ToLower(strIPFamily)
+	switch strIPFamily {
+	case "ipv4":
+		ipFamily = v1.IPv4Protocol
+	case "ipv6":
+		ipFamily = v1.IPv6Protocol
+	default:
+		return fmt.Errorf("unknown IPFamily: %s", strIPFamily)
```

Comment:
None of the other options do that, which would make this behavior inconsistent.

Reply:
It would make the behavior superior. Us look smarter and the user happier.

(Can't believe I'm saying this) I don't see how consistency here matters that much. All errors are including a string, this will be slightly more verbose. But maybe there are issues - the list getting out of sync?

Reply:
I got no issue against happier users; I got an issue with trying to piggy back that in this PR.

It will be done as a follow-up pr. OK with you ?

Reply:
Totally, thanks!